apps/src/javalab/JavalabButton.jsx

```diff
@@ -56,6 +66,12 @@ const styles = {
   horizontal: {
     flexDirection: 'row'
   },
+  disabled: {
+    backgroundColor: color.light_gray,
+    borderColor: color.light_gray,
+    ':hover': null,
+    cursor: 'default'
+  },
```

Comment:
Question: do we have a global "disabled" style somewhere that other apps also use, or is this specific to Javalab?

Reply:
mmm good question...I have not as much experience in our front end so I may not the best person to answer this, but I think the short answer is that it varies... 

We have this `Button` component which has [disabled styling built in](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/templates/Button.jsx#L281), but we are not using it in JavalabButton at the moment (listed as a to do elsewhere in this file).

It looks like we do have some [standard disabled styling](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/assets/stylesheets/application.scss#L3036) here for (HTML) buttons, which would probably work in this case?

The problem is that 3 of the 5 buttons I was trying to style are not `JavalabButton` but `PaneButton`, which are rendered as divs and I could not figure out how to apply disabled styling without my own custom styles.

Reply:
I see yeah, might be a big effort to consolidate all of these for now